README.md

```diff
@@ -73,7 +73,7 @@ Here's how to build the StableHLO repo on Linux or macOS:
 4. Make sure you check out the correct commit in the LLVM repository:
 
    ```sh
-   (hash="$(cat ../build_tools/llvm_version.txt)"; cd llvm-project && git fetch "$hash" && git checkout "$hash")
+   (hash="$(cat ./build_tools/llvm_version.txt)"; cd llvm-project && git fetch origin "$hash" && git checkout "$hash")
```

Comment:
Is `..` right?

I think we should be in repo root after

```
git clone --depth=1 --single-branch https://github.com/openxla/stablehlo
cd stablehlo
```

Reply:
Yeah, that’s why I changed `..` to `.`, since the previous command didn’t work.